ads/google/a4a/experiment-utils.js

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  */
 
+import {addExperimentIdToElement} from './traffic-experiments';
 import {
-  ExperimentInfo, // eslint-disable-line no-unused-vars
   forceExperimentBranch,
   getExperimentBranch,
   randomlySelectUnsetExperiments,
 } from '#experiments';
-import {addExperimentIdToElement} from './traffic-experiments';
+// Typedef imports
```

Comment:
Optional: Perhaps drop this comment since the `Def` type suffix and the `.type` file suffix make it obvious that this is a typedef?

Reply:
This has been a "convention" I've been introducing as I fix the type system in AMP code to vertically separate typedef imports, in part to make it easier to find commonly used type imports (for externing). I have WIP that lets the built-in eslint import rules sort/separate imports this way, but for now I'd like to leave this as-is if you don't feel too strongly about it.

Reply:
Makes sense. Carry on!